### PR TITLE
feat: add flag for disabling versions endpoint creation

### PR DESCRIPTION
**Pull request by ollz272**
## Pull Request Checklist

- [x] Make sure to read the contributor guide [here](https://github.com/alexschimpf/fastapi-versionizer/tree/main/CONTRIBUTE.md).
- [x] Make sure you are making a pull request against the main branch.
- [x] Make sure your pull request title matches the requested format.
- [x] Make sure to lint, type-check, and run unit and API tests.

## Description
We want to be able to customise the versions endpoint a bit but currently theres no way to disable it for us to do our own this. This PR adds the ability to enable/disable this endpoint

### CODE REVIEW

File: tests/test_simple_no_versions_route.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+from fastapi.testclient import TestClient
+
+from unittest import TestCase
+from examples.simple_no_versions_route import app, versions
+
+
+class TestSimple(TestCase):
```
Comment (alexschimpf) [inline tests/test_simple_no_versions_route.py:7]: The convention thus far has been to name the test class and test function similarly to the test filename

File: examples/simple_no_versions_route.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    - /v4/do_something
+    - /v4/do_something_else
+'''
+versions = versionize(
+    app=app,
+    prefix_format='/v{major}',
+    docs_url='/docs',
+    redoc_url='/redoc',
+    enable_version_route=False
+)
```
Comment (alexschimpf) [inline examples/simple_no_versions_route.py:89]: Need a newline at the end of the file

File: fastapi_versionizer/versionizer.py [hunk 3]
```diff
@@ -98,6 +98,7 @@ def versionize(
     get_docs: Union[Callable[[Tuple[int, int]], HTMLResponse], None] = None,
     get_redoc: Union[Callable[[Tuple[int, int]], HTMLResponse], None] = None,
     callback: Union[Callable[[FastAPI, Tuple[int, int], str], None], None] = None,
+    enable_version_route: bool = True,
```
Comment (alexschimpf) [inline fastapi_versionizer/versionizer.py:101]: Nitpick: Can we call it `enable_versions_route` instead?

File: examples/simple_no_versions_route.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+'''
+- Notes:
+    - "/v1/docs" and "/v2/docs" pages are generated, because `docs_url` is given
+    - "/v1/redoc" and "/v2/redoc" pages are generated because `redoc_url` is given
+    - "/versions" is automatically generated
+
+- This will create the following endpoints:
+    - /openapi.json
+    - /versions
```
Comment (alexschimpf) [inline examples/simple_no_versions_route.py:57]: This example should not create a "/versions" route

File: examples/simple_no_versions_route.py [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+@app.post('/do_something', tags=['Something'])
+async def do_something_v4() -> Any:
+    return {'message': 'something re-added'}
+
+
+'''
+- Notes:
+    - "/v1/docs" and "/v2/docs" pages are generated, because `docs_url` is given
+    - "/v1/redoc" and "/v2/redoc" pages are generated because `redoc_url` is given
+    - "/versions" is automatically generated
```
Comment (alexschimpf) [inline examples/simple_no_versions_route.py:53]: This example should not create a "/versions" route

**Comment by ollz272** [comment]
@alexschimpf Thanks! All done 👍🏻

**Review by alexschimpf — APPROVED** [review]

**Pull request merged**